swift/Samples/TicTacToe/Sources/Game/RunGameWorkflow.swift

```diff
@@ -84,19 +84,19 @@ extension RunGameWorkflow {
 // MARK: Rendering
 
 extension RunGameWorkflow {
-    typealias Rendering = AlertContainerScreen<ModalContainerScreen<BackStackScreen>>
+    typealias Rendering = AlertContainerScreen<ModalContainerScreen<BackStackScreen<AnyScreen>>>
```

Comment:
😮

Reply:
Yeah…

Reply:
What's the 😮 ?

Reply:
Definitely the right direction as far as type-safety is concerned. Not ideal from the API ergonomics perspective. However, once we bump up to iOS 13 we'll be able to use [opaque types](https://docs.swift.org/swift-book/LanguageGuide/OpaqueTypes.html) and this won't be a concern.

Reply:
I think it's a comment on the # of generics `A<B<C<D>>>`.